packages/website/src/components/Hero/index.tsx

```diff
@@ -1,32 +1,67 @@
 import React from "react";
 
 export default function Hero() {
+  // Enlarges and decreases the opacity of the taikoGeom image upon scroll
+  const changeTaikoGeom = () => {
+    const taikoGeom = document.getElementById("taikoGeom");
+    const taikoGeomParent = document.getElementById("taikoGeomParent");
+    const elementHeight = window.pageYOffset;
+
+    // Enlarge the TaikoGeom width on scroll
+    if (window.innerWidth.valueOf() < 500) {
+      taikoGeomParent.style.width = `${
+        window.innerWidth.valueOf() / 1 + elementHeight * 9
+      }px`;
+
+      // Lower the TaikoGeom opacity on scroll
+      // Smaller screens need a faster decrease
+      if (1 - elementHeight * 0.003 >= 0) {
+        taikoGeom.style.opacity = `${
+          (1 - elementHeight * 0.003) * (1 - elementHeight * 0.003) * 0.8
+        }`;
+      }
+    } else {
+      taikoGeomParent.style.width = `${
+        window.innerWidth.valueOf() / 2 + elementHeight * 9
+      }px`;
+
+      // Lower the TaikoGeom opacity on scroll
+      taikoGeom.style.opacity = `${(1 - elementHeight / 250) * 0.8}`;
+    }
+  };
+  window.addEventListener("scroll", changeTaikoGeom);
```

Comment:
Does this need to ever be removed at any point?

Reply:
This is the animation feature on the taiko geometry image that I played around with, I like the outcome but if others don't like it that section could be removed

Reply:
Actually to me it looks good but was just wondering about the listener, usually memory leaks occur if it isn't removed but not sure about that with svelte.

Reply:
this is not using svelte fyi, it is using react. (we use svelte on our Bridge UI, this is docusaurus + React). @WormydogDOTeth